html/index.html

```diff
@@ -3,9 +3,213 @@
 <head>
   <meta charset="UTF-8">
   <title>Document</title>
+  <link href='css/less/styles.css' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css?family=Raleway:700|Montserrat&display=swap" rel="stylesheet">
 </head>
 <body>
-  
-    <script src="http://localhost:35729/livereload.js"></script>
+  <nav>
+    <a href='#' id='home_link'>Adam Dow</a>
+    <a href='#'>Improv</a>
+    <a href='#'>Services</a>
+    <a href='#'>Gallery</a>
+    <a href='#'>About</a>
+    <a href='#'>Contact</a>
+  </nav>
+  <div>
```

Comment:
this first `<div>` will be a slideshow of 3 slides. You should look up a responsive slideshow plugin to use so you can write code using the proper syntax.